NineChronicles.Headless/GraphTypes/States/Models/Table/MonsterCollectionSheetType.cs

```diff
@@ -13,9 +14,12 @@ public MonsterCollectionSheetType()
                 nameof(MonsterCollectionSheet.OrderedList),
                 resolve: context =>
                 {
-                    return context.Source.monsterCollectionSheet.OrderedList
-                        .Select(r => (r, context.Source.monsterCollectionRewardSheet))
-                        .ToList();
+                    return context.Source.monsterCollectionSheet.OrderedList?
+                               .Select(r => (r, context.Source.monsterCollectionRewardSheet))
+                               .ToList()
+                           ?? new List<(
+                               MonsterCollectionSheet.Row r,
+                               MonsterCollectionRewardSheet monsterCollectionRewardSheet)>();
```

Comment:
It seems okay to return null because the type is `ListGraphType<MonsterCollectionRowType>`.